shared/cfg/machines.cfg

```diff
@@ -41,6 +41,7 @@ variants:
         nvdimm_uuid = <auto>
     - arm64-mmio:
         only aarch64
+        only ovmf
```

Comment:
I'm afraid this change would cause the failure for running any upstream testing since the default variant of firmware is `default_bios`.

Reply:
Yep, agree with you.

The easiest way is to move related parameters from ovmf to default_bios since they are required by aarch64. Otherwise, need to change the default filter from default_bios to ovmf for aarch64.

@luckyh What do you think?

Reply:
Or, we could make it work for both (obviously a bad effect by the ill-defined variants.)

Reply:
Moved them to `machines.cfg`